### ISSUE TITLE
[Suggestion] Persistent collapsible headings

### ISSUE THREAD

**Issue by octanervoc** [cqroot/joplin-outline#89]
Currently, whenever the user edits the current note by writing a new line, all headings in the TOC panel expand. The animation below illustrates this:

![peek_2_noloop](https://github.com/cqroot/joplin-outline/assets/98230395/bd117ba5-3a75-400f-98aa-8f3bd3d853dd)

It would be very useful to add an option to the plugin of maintaining the collapsible state of the headings. In very long notes, with many different headers, it's cumbersome to always have to re-collapse in the TOC all headers not currently in use.

Thank you for your work on the plugin. It drastically improves productivity on the app.

**Comment by knas4ever** [comment]
yes this, or just a option to have the Outline collapsed as default when opening a note.
